ontology/envo.md

```diff
@@ -42,40 +42,32 @@ dependencies:
   - id: po
   - id: go
 usages:
-  - type: data-annotation
+  - user: http://eol.org
+    type: data-annotation
     description: "describing species habitats"
     examples:
-      url: http://eol.org/pages/211700/data
-    resources:
-      url: http://eol.org
-      label: EOL
-  - type: data-annotation
+      - url: http://eol.org/pages/211700/data
+        description: "Pseudobarbus burchelli (Tradou Redfin) is a species of bony fishes in the family Cyprinidae. They are associated with freshwater habitat. Individuals can grow to 13.5 cm. They have sexual reproduction."
+  - user: http://globalbioticinteractions.org
+    type: data-annotation
     description: "describing stomach contents"
-    datasets:
-      url: https://s3.amazonaws.com/globi/snapshot/target/data/taxa/interactions.csv.gz
```

Comment:
This is getting dropped, but I think that's OK since it's not a very good example.